lib/IHP/Makefile.dist

```diff
@@ -114,10 +114,10 @@ psql: ## Connects to the running postgresql server
 	@psql -h $$PWD/build/db -d app
 
 db: Application/Schema.sql Application/Fixtures.sql ## Creates a new database with the current Schema and imports Fixtures.sql
-	echo "drop schema public cascade; create schema public;" | psql -h $$PWD/build/db -d app
-	psql -h $$PWD/build/db -d app < "${IHP_LIB}/IHPSchema.sql"
-	psql -h $$PWD/build/db -d app < Application/Schema.sql
-	psql -h $$PWD/build/db -d app < Application/Fixtures.sql
+	(echo "drop schema public cascade; create schema public;" | psql -h $${PWD}/build/db -d app) && \
```

Comment:
Looking at the other make files, we don't use `&& \`

Reply:
Yep, but this way, we can stop on error. Otherwise it goes on, as each line is executed in a new subshell.